client/src/components/ClassCard.jsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { Card, Button } from 'semantic-ui-react';
+
+import { putUserClass } from '../utils/apiWrapper';
+
+// import '../css/Class.scss';
+
+function ClassCard(props) {
+  const { text } = props;
```

Comment:
This variable name is not descriptive enough. `Text` is ambiguous; give it a more descriptive name that describes what the text is